.github/workflows/ci.yml

```diff
@@ -19,17 +19,113 @@ jobs:
+      - run: composer update --ansi --no-progress --prefer-dist --no-interaction
+      - run: vendor/bin/phpstan analyze --ansi --no-progress
+
+  Official:
+    name: "🛡️ ${{ matrix.name }}"
+    uses: "./.github/workflows/test.yml"
+    with:
+      name: ${{ matrix.name }}
+      driverRepoUrl: ${{ matrix.driverRepoUrl }}
+      php: ${{ matrix.php }}
+      setUpCmd: ${{ matrix.setUpCmd }}
+      testCmd: ${{ matrix.testCmd }}
+      tearDownCmd: ${{ matrix.tearDownCmd }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "BrowserKit (http client)"
+            driverRepoUrl: "https://github.com/minkphp/MinkBrowserKitDriver.git"
+            php: "7.2"
+            # language=bash
+            testCmd: ./vendor/bin/phpunit --colors=always --testdox --configuration ./phpunit.http_client.xml
+
+          - name: "BrowserKit (http kernel)"
+            driverRepoUrl: "https://github.com/minkphp/MinkBrowserKitDriver.git"
+            php: "7.2"
+            # language=bash
+            testCmd: ./vendor/bin/phpunit --colors=always --testdox
+
+          - name: "Selenium2"
+            driverRepoUrl: "https://github.com/minkphp/MinkSelenium2Driver.git"
+            php: "7.2"
+            # language=bash
+            setUpCmd: |
+              export SELENIUM_IMAGE=selenium/standalone-firefox:2.53.1
+              docker compose up --wait --quiet-pull
+              curl --retry 5 --retry-all-errors --retry-delay 1 --max-time 10 --head -X GET http://localhost:4444/wd/hub/status
+            # language=bash
+            testCmd: |
+              export WEB_FIXTURES_BROWSER=firefox
+              export DRIVER_MACHINE_BASE_PATH=/fixtures/
+              export WEB_FIXTURES_HOST=http://host.docker.internal:8002
+              ./vendor/bin/phpunit --colors=always --testdox
+            # language=bash
+            tearDownCmd: |
+              docker compose logs --no-color --no-log-prefix --timestamps selenium > ./logs/selenium.docker.log
+
+          - name: "WebDriver-Classic"
+            driverRepoUrl: "https://github.com/minkphp/webdriver-classic-driver.git"
+            php: "7.4"
+            # language=bash
+            setUpCmd: |
+              export SELENIUM_IMAGE=selenium/standalone-firefox:4
+              docker compose up --wait --quiet-pull
+              curl --retry 5 --retry-all-errors --retry-delay 1 --max-time 10 --head -X GET http://localhost:4444/wd/hub/status
+            # language=bash
+            testCmd: |
+              export WEB_FIXTURES_BROWSER=firefox
+              export DRIVER_MACHINE_BASE_PATH=/fixtures/
+              export WEB_FIXTURES_HOST=http://host.docker.internal:8002
+              ./vendor/bin/phpunit --colors=always --testdox
+            # language=bash
+            tearDownCmd: |
+              docker compose logs --no-color --no-log-prefix --timestamps selenium > ./logs/selenium.docker.log
+
+  Unofficial:
+    name: "✨️ ${{ matrix.name }}"
```

Comment:
I just pushed a major update; it should work much better now and should be relatively easy to follow.

There's one major annoyance (on GitHub's side): they do not evaluate expressions in `job.name` (that line above) when the job is skipped.

This causes some ugly UI:
![image](https://github.com/user-attachments/assets/13d827b6-220a-4017-9e41-2559d8418b1e)

- https://github.com/orgs/community/discussions/13261
- https://github.com/actions/runner/issues/1215

Reply:
A way to solve that would be to put those jobs in a separate workflow instead (that can be dispatched manually) instead of putting them in the `CI` workflow but skipping them anytime it is not dispatched manually. This would not cause duplication, as the common logic is already in the reusable workflow anyway.

Reply:
Good idea, I'll try implementing it so.

Reply:
@stof I tried it out on my personal branch and it looked like it works really nicely. Can't see any downsides so far, in fact (except that the current "CI" entry will become obsolete, but I think that's really minor).